src/magic.cpp

```diff
@@ -199,6 +199,8 @@ void spell_type::load( JsonObject &jo, const std::string & )
     mandatory( jo, was_loaded, "id", id );
     mandatory( jo, was_loaded, "name", name );
     mandatory( jo, was_loaded, "description", description );
+    auto default_msg = translation( "You cast %s!" );
```

Comment:
```suggestion
    translation default_msg = translation( "You cast %s!" );
```
it is preferred that we avoid `auto` unless it's an iterator or some other similar complex type